manifests/server.pp

```diff
@@ -104,17 +138,14 @@
   contain ssh::server::config
   contain ssh::server::service
 
-  # Provide option to *not* use storeconfigs/puppetdb, which means not managing
-  #  hostkeys and knownhosts
-  if ($storeconfigs_enabled) {
+  # Provide option to *not* use storeconfigs/puppetdb, which means not exporting hostkeys
+  if $storeconfigs_enabled {
     contain ssh::hostkeys
-    contain ssh::knownhosts
 
     Class['ssh::server::install']
     -> Class['ssh::server::config']
     ~> Class['ssh::server::service']
     -> Class['ssh::hostkeys']
-    -> Class['ssh::knownhosts']
```

Comment:
I think there's a chance, that someone's making use of this by not managing the client, but still making use of the collected known hosts. Except for getting rid of the class, is there any disadvantage which gets solved?